requirements.txt

```diff
@@ -1,5 +1,3 @@
-# The following line refers to a private repository, so it requires a
-# GitHub token for installation
-# https://github.com/jsf9k/cyhy-core/tarball/develop
-dateutil
 docopt
+python-dateutil
+https://github.com/cisagov/mongo-db-from-config/tarball/develop
```

Comment:
It's best to sort these alphabetically by the name of the package.  So the URL line should go above python-dateutil.

Reply:
I love a good nit to pick!  Fixed in dec752d. Thanks for the keen eye.